jupyter/internal/setup-jupyter-kernel.sh

```diff
@@ -15,30 +15,34 @@ JUPYTER_PORT=$(/usr/share/google/get_metadata_value attributes/JUPYTER_PORT || t
 [[ ! $JUPYTER_PORT =~ ^[0-9]+$ ]] && JUPYTER_PORT=8123
 readonly JUPYTER_PORT
 readonly JUPYTER_AUTH_TOKEN="$(/usr/share/google/get_metadata_value attributes/JUPYTER_AUTH_TOKEN || true)"
-readonly JUPYTER_KERNEL_DIR='/dataproc-initialization-actions/jupyter/kernels/pyspark'
-readonly KERNEL_GENERATOR='/dataproc-initialization-actions/jupyter/kernels/generate-pyspark.sh'
-readonly TOREE_INSTALLER='/dataproc-initialization-actions/jupyter/kernels/install-toree.sh'
+
+readonly LOCAL_INIT_ACTIONS_REPO=/tmp/local-initialization-actions
+readonly JUPYTER_KERNEL_DIR="${LOCAL_INIT_ACTIONS_REPO}/jupyter/kernels/pyspark"
+readonly KERNEL_GENERATOR="${LOCAL_INIT_ACTIONS_REPO}/jupyter/kernels/generate-pyspark.sh"
+readonly TOREE_INSTALLER="${LOCAL_INIT_ACTIONS_REPO}/jupyter/kernels/install-toree.sh"
 
 [[ "${ROLE}" != 'Master' ]] && throw "${0} should only be run on the Master node!"
 
 hadoop fs -mkdir -p "gs://${NOTEBOOK_DIR}"
 
 echo "Creating Jupyter config..."
 jupyter notebook --allow-root --generate-config -y --ip=127.0.0.1
-echo "c.Application.log_level = 'DEBUG'" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.NotebookApp.ip = '0.0.0.0'" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.NotebookApp.open_browser = False" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.NotebookApp.port = ${JUPYTER_PORT}" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.NotebookApp.contents_manager_class = 'jgscm.GoogleStorageContentManager'" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.GoogleStorageContentManager.default_path = '${NOTEBOOK_DIR}'" >> ~/.jupyter/jupyter_notebook_config.py
-echo "c.NotebookApp.token = u'${JUPYTER_AUTH_TOKEN}'" >> ~/.jupyter/jupyter_notebook_config.py
+
+cat <<EOF >>~/.jupyter/jupyter_notebook_config.py
+c.Application.log_level = 'DEBUG'
+c.NotebookApp.ip = '0.0.0.0'
```

Comment:
Add an empty line before this line to handle the case that there is no newline character at the end of that file.

Reply:
Done.